src/Experiment/src/AssemblyBuilder.cs

```diff
@@ -5,10 +5,12 @@
 using System.Threading.Tasks;
 using System.Reflection;
 
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
 
 namespace System.Reflection.Emit.Experimental
 {
-    class AssemblyBuilder: System.Reflection.Assembly
+   public class AssemblyBuilder: System.Reflection.Assembly
```

Comment:
```suggestion
    public class AssemblyBuilder: System.Reflection.Assembly
```
NIT: missing a space in the indentation